src/components/ChatCustomAvatar.js

```diff
@@ -0,0 +1,63 @@
+import React, {PureComponent} from 'react';
+import {Image, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import ActiveRoomAvatar from '../../assets/images/avatars/room.svg';
+
+const propTypes = {
+    /** Url source for the avatar */
+    source: PropTypes.string,
+
+    /** Extra styles to pass to Image */
+    imageStyles: PropTypes.arrayOf(PropTypes.object),
+
+    /** Extra styles to pass to View wrapper */
+    containerStyles: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object)]),
+
+    /** Set the size of ChatCustomAvatar */
+    size: PropTypes.oneOf(['default', 'small']),
+
+    /** Whether this avatar is for an custom room */
+    isCustomChatRoom: PropTypes.bool,
+
+    /** Whether this avatar is for an custom room */
+    CustomChatRoomIcon: PropTypes.func,
+
+};
+
+const defaultProps = {
+    source: '',
+    imageStyles: [],
+    containerStyles: [],
+    size: 'default',
+    isCustomChatRoom: false,
+    CustomChatRoomIcon: ActiveRoomAvatar,
+
+
+};
+
+class ChatCustomAvatar extends PureComponent {
```

Comment:
I don't feel the urge to use PureComponent.

Reply:
and got this error while linting, when I converted PureComponent to only Component.

This is a good candidate for the Functional component.

Reply:
Please convert it to the functional component and add don't forget to add `displayName` https://github.com/Expensify/App/blob/main/STYLE.md#stateless-component-style.

Reply:
converted